src/js/edu-benefits/utils/veteran.js

```diff
@@ -47,6 +54,8 @@ export function createVeteran() {
     toursOfDuty: [],
     seniorRotcComissioned: makeField(''),
     seniorRotcComissionYear: makeField(''),
+    seniorRotcScholarships: [],
```

Comment:
I think we want to match the json schema here if possible: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/dist/edu-benefits-schema.json#L450